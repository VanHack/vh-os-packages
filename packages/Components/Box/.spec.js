import React from 'react';
import VHBox from '.';
import { render, renderer } from '../../../setupTests';

describe('[Component] - VHBox', () => {
  it('renders box without crashing', () => {
    const { getByText } = render(<VHBox>Box</VHBox>);
    expect(getByText('Box')).toMatchSnapshot();
  });

  it('renders vhbox with bgColor and color', () => {
    const json = renderer(
      <VHBox bg="#e99" color="#ccc">
        Box
      </VHBox>
    ).toJSON();
    expect(json).toHaveStyleRule('background-color', '#e99');
    expect(json).toHaveStyleRule('color', '#ccc');
  });
});
