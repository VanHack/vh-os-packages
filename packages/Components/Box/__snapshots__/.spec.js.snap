// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Component] - VHBox renders box without crashing 1`] = `
<div
  class=""
>
  Box
</div>
`;
